daemon/network.go

```diff
@@ -468,3 +468,35 @@ func (daemon *Daemon) deleteNetwork(networkID string, dynamic bool) error {
 func (daemon *Daemon) GetNetworks() []libnetwork.Network {
 	return daemon.getAllNetworks()
 }
+
+// clearAttachableNetworks removes the attachable networks
+// after disconnecting any connected container
+func (daemon *Daemon) clearAttachableNetworks() {
+	var networks []libnetwork.Network
+	for _, n := range daemon.GetNetworks() {
```

Comment:
Thanks, it in fact looks like `Getnetworks()` returns a new slice of networks.
Then there is no need to create a new slice here. 
Will fix it shortly. Thanks.